src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -92,10 +95,6 @@ with_halo(halo, ibg::ImmersedBoundaryGrid) =
 inflate_halo_size_one_dimension(req_H, old_H, _, ::IBG)            = max(req_H + 1, old_H)
 inflate_halo_size_one_dimension(req_H, old_H, ::Type{Flat}, ::IBG) = 0
 
-# Defining the bottom
-@inline z_bottom(i, j, grid) = znode(i, j, 1, grid, c, c, f)
-@inline z_bottom(i, j, ibg::IBG) = error("The function `bottom` has not been defined for $(summary(ibg))!")
```

Comment:
isn't this still useful? Why delete it?

Reply:
I have moved it in the `TurbulenceClosures` module where also `z_top` and `height_above_bottom` are defined

Reply:
Ok so its unrelated to the static column depth? I feel z_bottom belongs in the Grids module. What do you think?

Reply:
Consider that it has to be extended for different immersed boundary implementation so it depends on partial cell bottom, etc. 

The original implementation may not have been optimal, but since we are moving things around here, I think we should try to improve on the original organization.

Reply:
I am ok with moving all these functions in the Grids module, shall we do it in this PR?